nano/node/wallet.cpp

```diff
@@ -75,6 +75,19 @@ nano::public_key nano::wallet_store::deterministic_insert (nano::transaction con
 	return result;
 }
 
+nano::public_key nano::wallet_store::deterministic_insert (nano::transaction const & transaction_a, uint32_t const index)
+{
+	auto curr_index (deterministic_index_get (transaction_a));
```

Comment:
Seems not used

Reply:
Indeed, those were testing leftovers. An overloaded function seemed better at the end. Same for the other one, both removed.